modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -65,16 +65,39 @@ spec:
 <4> Specify the `kafka` type.
 <5> Specify the URL and port of the Kafka broker as a valid absolute URL, optionally with a specific topic. You can use the `tcp` (insecure) or `tls` (secure TCP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
```

Comment:
Oscar, this line mentions the `tcp` and `tls` prefixes. Should we replace `tcp` with `http` here. Or replace `http` on line 68 with `tcp`?